modules/client-configuration/pages/supported-features-centos.adoc

```diff
@@ -1,13 +1,13 @@
 [[supported-features-centos]]
-= Supported CentOS Features
+= Suppored CentOS Features
 
 
 This table lists the availability of various features on {CentOS} clients.
 
 [WARNING]
 ====
-{centos} is not currently an officially supported client operating system.
-It may or may not be supported in a future version of {productname}.
+Plain {centos} is not currently an officially supported client operating system. It may or may not be supported in a future version of {productname}.
```

Comment:
```suggestion
{centos} is not currently an officially supported client operating system. 
It may or may not be supported in a future version of {productname}.
```

I don't think we can use "plain" because it's not actually a CentOS version.

Reply:
Maybe "CentOS without Expanded Support is not currently..." ?